frontends/ol-components/package.json

```diff
@@ -16,11 +16,11 @@
     "@dnd-kit/utilities": "^3.2.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/base": "5.0.0-beta.60",
-    "@mui/lab": "^5.0.0-alpha.172",
-    "@mui/material": "^5.16.1",
-    "@mui/material-nextjs": "^5.16.6",
-    "@mui/system": "^5.16.1",
+    "@mui/base": "5.0.0-beta.61",
+    "@mui/lab": "6.0.0-beta.14",
+    "@mui/material": "^6.1.6",
+    "@mui/material-nextjs": "^6.1.6",
+    "@mui/system": "^6.1.6",
```

Comment:
I would be inclined to do this separately. Seems to have gone smoothly, but a major bump of a significant package.

I'd also be curious the impact on our package size. I think v6 is supposed to be a good bit smaller.

Reply:
@mui/material-nextjs@6.1.6 is needed for Next.js v15 and others were in the same [material-ui release](https://github.com/mui/material-ui/releases) (except @mui/base)